src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -54,15 +54,15 @@ public class JwtSecurityTokenHandler : SecurityTokenHandler, ISecurityTokenValid
         // Summary:
         //     The claim properties namespace.
         private const string Namespace = "http://schemas.xmlsoap.org/ws/2005/05/identity/claimproperties";
-        private static IDictionary<string, string> inboundClaimTypeMap = ClaimTypeMapping.InboundClaimTypeMap;
-        private static IDictionary<string, string> outboundClaimTypeMap = ClaimTypeMapping.OutboundClaimTypeMap;
         private static string shortClaimTypeProperty = Namespace + "/ShortTypeName";
         private static string jsonClaimTypeProperty = Namespace + "/json_type";
-        private static ISet<string> inboundClaimFilter = ClaimTypeMapping.InboundClaimFilter;
         private static string[] tokenTypeIdentifiers = { JwtConstants.TokenTypeAlt, JwtConstants.TokenType };
         private SignatureProviderFactory signatureProviderFactory = new SignatureProviderFactory();
         private Int32 _maximumTokenSizeInBytes = TokenValidationParameters.DefaultMaximumTokenSizeInBytes;
         private Int32 _defaultTokenLifetimeInMinutes = DefaultTokenLifetimeInMinutes;
+        private IDictionary<string, string> _inboundClaimTypeMap = DefaultInboundClaimTypeMap;
```

Comment:
Since these are by reference, two instances of JwtSecurityTokenHandler will point to the same instance of the Dictionary. The constructor will need to create a new Dictionary<string, string> for each instance. 
For example, create two JwtSecurityTokenHandlers j1, j2. call j1.InboundClaimTypeMap.Clear(). j2 will also have been cleared.